en/assembly/quick_start_pixhawk4.md

```diff
@@ -155,13 +163,13 @@ General configuration information is covered in: [Autopilot Configuration](../co
 
 QuadPlane specific configuration is covered here: [QuadPlane VTOL Configuration](../config_vtol/vtol_quad_configuration.md)
 
-<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? -->
+<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? --> 
 
-<!--## Detailed Wiring Infographic (Copter)
+<!--## Detailed Wiring Infographic (Copter) 

```

Comment:
Yes, it would be great to get detailed wiring infographics for all out our vehicles. We actually have NO great examples anywhere of VTOL or plane wiring (maybe a few of the build logs are OK).